pyhf/__init__.py

```diff
@@ -1,8 +1,9 @@
-import logging
 import pyhf.optimize as optimize
 import pyhf.tensor as tensor
 from . import exceptions
 
+import logging
+
```

Comment:
can we move this up again :) I think it's good to have the stdlib imports on top

Reply:
Absolutely! I will try to make a mental note of this and always put those imports first. :+1: